### PR TITLE
Fixed name search

### PR DESCRIPTION
**Pull request by TheMiloAnderson**
Made mocList a global variable, and assign it initially at get116thCongress(), so that it's available in the name search functions.

Added some extra card divs with 0 height, so the widths are consistent in different layouts

### CODE REVIEW

**Comment by meganrm** [comment]
we don't want this to be global, because if the person has selected either of the two tabs, the list should be filtered. it's supposed to be locally scoped. If you have Senate selected and search for a rep it is fine that nobody comes back. Also please pull in the most recent changes from master.

**Comment by TheMiloAnderson** [comment]
That's why I made it global, though, so that the house/senate filtering can work in conjunction with the search. I could've passed the global MoCs variable to addMoCCards() in the search handler functions, but if I did the search would always run with the full MoC list, not the filtered house/senate list.

**Comment by TheMiloAnderson** [comment]
Just noticed you have:

addMoCCards(selectedTab === FULL_CONGRESS ? MoCs : MoCs.filter(moc => moc.chamber === selectedTab));

...in the filter functions below. That works too! I've updated to match...

**Comment by meganrm** [comment]
yeah, i would prefer we move away from using globals in here. it can be really difficult to follow.

**Comment by meganrm** [comment]
The diff still looks weird. it's adding back things I've changed in master.

File: src/script.js [hunk 1]
```diff
@@ -133,6 +134,62 @@ $('.scroll-link').on('click', (e) => {
     scrollToAnchor(link)
 })
 
+function makeRow(name, status){
```
Comment (meganrm) [inline src/script.js]: i've moved this into a `map/tooltip` file

File: src/script.js [hunk 2]
```diff
@@ -29,7 +29,7 @@ $('.congress-toggle a').on('click', function (e) {
   $(this).addClass('active');
   let newSelectedTab = $(this).attr('data-value');
   if (newSelectedTab !== selectedTab) {
-    let mocList = newSelectedTab === FULL_CONGRESS ? MoCs : MoCs.filter((moc) => moc.chamber === newSelectedTab);
+    const mocList = newSelectedTab === 'full' ? MoCs : MoCs.filter((moc) => moc.chamber === newSelectedTab);
```
Comment (meganrm) [inline src/script.js]: this should be `FULL_CONGRESS` and not 'full'

**Pull request closed without merging**